src/Sarif.Multitool.Library/Rules/RuleId.cs

```diff
@@ -49,6 +49,17 @@ public static class RuleId
         public const string ProvideCheckoutPath = "GH1006";
         public const string ProvideFullyFormattedMessageStrings = "GH1007";
 
+        // Rules for GHAS.
+        public const string GHASReferenceFinalSchema = "GH1011";
```

Comment:
>GHAS [](http://example.com/codeflow?start=28&length=4)

Should we change these as well?

Reply:
got it

Reply:
But if change it, does it mean we also change this:

![image](https://github.com/microsoft/sarif-sdk/assets/81775155/0925afcc-faca-4ff5-881c-463c9099d2a0)

it may trigger a few more changes for consistency.

Reply:
Never mind, leave these alone. We can change them later if necessary